evals/registry/evals/rajini-trivia.yaml

```diff
@@ -0,0 +1,8 @@
+rajini-trivia:
+  id: rajini-trivia.dev-v0
+  description: Is GPT-4 better thn an average Rajini fan?
```

Comment:
you have a typo here :0 thn instead of than

Reply:
fixed, thanks.